crates/module-system/sov-modules-api/src/transaction/mod.rs

```diff
@@ -70,6 +75,127 @@ pub struct Version0<Call, S: Spec> {
     pub details: TxDetails<S>,
 }
 
+#[derive(
+    derive_more::Debug,
+    Clone,
+    borsh::BorshDeserialize,
+    serde::Serialize,
+    serde::Deserialize,
+    borsh::BorshSerialize,
+    UniversalWallet,
+    PartialEq,
+    Eq,
+)]
+/// A signature and public key pair.
+pub struct PubKeyAndSignature<S: Spec> {
```

Comment:
You not always need the public key as it can be recovered from the sig and the message. Will save you some precious bytes